### DISCUSSION TITLE
[Question] Only show containers in workspace

### DISCUSSION THREAD

**Discussion by lonix1** [microsoft/vscode-docker#3008] [Q&A] [answered]
My workspace contains some `Dockerfile` and `docker-compose.yml` files. I'd like to be able to manage those using the extension.

However the extension shows all running containers, even those defined outside my workspace.

Is it possible to restrict the extension to show only those containers defined in the workspace?

**Comment by karolz-ms** [comment, marked as answer]
Good question @lonix1 As of now there is no straightforward way to restrict what Docker view displays to show only containers and images related to a workspace. That said, naming and labeling your containers and your Docker Compose files should help. For example, by default containers launched from the same Compose file ("Compose project") are grouped together in containers pane. 

All the panes in the Docker view allow you to customize the grouping and sorting settings--just click on the "Configure Explorer" ("gear") icon and see what is available.

Hope this helps!

**Reply by lonix1** [reply]
Thanks I'll try that!
